src/BoilerPlate/App.cpp

```diff
@@ -144,31 +201,34 @@ namespace Engine
 
 	void App::Render()
 	{
-		//            r      g    y		b
-		glClearColor(0.0, 0.50f, 0.50f, 1.0f);
+		//            r      g    b		a
+		glClearColor(0.10, 0.15f, 0.40f, 1.0f);
 		glClear(GL_COLOR_BUFFER_BIT);
 
-		glBegin(GL_LINE_LOOP);
-		glColor3f(1.0f, 0.55f, 0.50f);
-		glVertex2f(ejeXp, ejeYp);
-		glVertex2f(ejeXp, ejeYn);
-		glVertex2f(ejeXn, ejeYp);
-		glVertex2f(ejeXn, ejeYn);
-		glEnd();
+		playerONE.Render();
+		gameUtility.RenderGalaxy();
+		
+		if (gameUtility.deBuggtool) { //if the debug tool is on.
+			playerONE.DrawHollowCircle();
+			gameUtility.DrawAsteroidCircles();
+			for (int i = 0; i < gameUtility.Galaxy.size(); i++)
```

Comment:
This is screaming for a refactor, start by moving stuff to a function, then try to improve the design of your implementation